content/ko/guides/models/sweeps/start-sweep-agents.md

```diff
@@ -1,90 +1,88 @@
+`wandb agent` 코맨드를 사용하여 스윕을 시작하세요. 스윕을 초기화할 때 반환된 스윕 ID를 제공하세요. 아래 코드 조각을 복사하여 붙여넣고 `sweep_id`를 스윕 ID로 바꾸세요.
 
 ```bash
 wandb agent sweep_id
 ```
 {{% /tab %}}
 {{% tab header="Python script or notebook" %}}
-Use the W&B Python SDK library to start a sweep. Provide the sweep ID that was returned when you initialized the sweep. In addition, provide the name of the function  the sweep will execute.
+W&B Python SDK 라이브러리를 사용하여 스윕을 시작하세요. 스윕을 초기화할 때 반환된 스윕 ID를 제공하세요. 또한 스윕이 실행할 함수의 이름을 제공하세요.
 
 ```python
 wandb.agent(sweep_id=sweep_id, function=function_name)
 ```
 {{% /tab %}}
 {{< /tabpane >}}
 
-
-
-### Stop W&B agent
+### W&B 에이전트 중지
 
 {{% alert color="secondary" %}}
-Random and Bayesian searches will run forever. You must stop the process from the command line, within your python script, or the [Sweeps UI]({{< relref path="./visualize-sweep-results.md" lang="ko" >}}).
+랜덤 및 베이지안 탐색은 영원히 실행됩니다. 코맨드 라인, Python 스크립트 또는 [Sweeps UI]({{< relref path="./visualize-sweep-results.md" lang="ko" >}}) 내에서 프로세스를 중지해야 합니다.
 {{% /alert %}}
 
-Optionally specify the number of W&B Runs a Sweep agent should try. The following code snippets demonstrate how to set a maximum number of [W&B Runs]({{< relref path="/ref/python/run.md" lang="ko" >}}) with the CLI and within a Jupyter Notebook, Python script.
+선택적으로 스윕 에이전트가 시도해야 하는 W&B Runs의 수를 지정하세요. 다음 코드 조각은 CLI 및 Jupyter Notebook, Python 스크립트 내에서 최대 [W&B Runs]({{< relref path="/ref/python/run.md" lang="ko" >}}) 수를 설정하는 방법을 보여줍니다.
 
 {{< tabpane text=true >}}
   {{% tab header="Python script or notebook" %}}
-First, initialize your sweep. For more information, see [Initialize sweeps]({{< relref path="./initialize-sweeps.md" lang="ko" >}}).
+먼저 스윕을 초기화합니다. 자세한 내용은 [스윕 초기화]({{< relref path="./initialize-sweeps.md" lang="ko" >}})을 참조하세요.
 
 ```
 sweep_id = wandb.sweep(sweep_config)
 ```
 
-Next, start the sweep job. Provide the sweep ID generated from sweep initiation. Pass an integer value to the count parameter to set the maximum number of runs to try.
+다음으로 스윕 작업을 시작합니다. 스윕 시작에서 생성된 스윕 ID를 제공합니다. 시도할 최대 run 수를 설정하려면 count 파라미터에 정수 값을 전달하세요.
 
 ```python
 sweep_id, count = "dtzl1o7u", 10
 wandb.agent(sweep_id, count=count)
 ```
 
 {{% alert color="secondary" %}}
-If you start a new run after the sweep agent has finished, within the same script or notebook, then you should call `wandb.teardown()` before starting the new run.
-{{% /alert %}}  
+스윕 에이전트가 완료된 후 동일한 스크립트 또는 노트북 내에서 새 run을 시작하는 경우 새 run을 시작하기 전에 `wandb.teardown()`을 호출해야 합니다.
+{{% /alert %}}
   {{% /tab %}}
   {{% tab header="CLI" %}}
-First, initialize your sweep with the [`wandb sweep`]({{< relref path="/ref/cli/wandb-sweep.md" lang="ko" >}}) command. For more information, see [Initialize sweeps]({{< relref path="./initialize-sweeps.md" lang="ko" >}}).
+먼저 [`wandb sweep`]({{< relref path="/ref/cli/wandb-sweep.md" lang="ko" }}) 코맨드로 스윕을 초기화합니다. 자세한 내용은 [스윕 초기화]({{< relref path="./initialize-sweeps.md" lang="ko" >}})을 참조하세요.
```

Comment:
Removing this line, link enables the docs to build (locally)...not sure why though....
```suggestion
[`wandb sweep`]({{< relref path="/ref/cli/wandb-sweep.md" lang="ko" }})
```